### PR TITLE
Fixed deprecated link in certifications-and-compliance.rst

### PR DESCRIPTION
**Pull request by ArmanChand**

### CODE REVIEW

File: source/about/certifications-and-compliance.rst [hunk 1]
```diff
@@ -55,22 +55,22 @@ Mattermost supports features that ensure data management and data portability.
 Data Management
 ^^^^^^^^^^^^^^^^
 
-- **Data Retention:** Use `data retention <https://docs.mattermost.com/administration/data-retention.html>`__ to automatically erase data after a set period of time, a feature that meets the Right to Erasure principle. In Team Edition, you can use database scripts to achieve the same result.
-- **Profile Deletion:** Delete a user’s personal information via `a command line tool <https://docs.mattermost.com/administration/command-line-tools.html#platform-user-delete>`__. The command permanently deletes all user information including messages created by the user.
+- **Data Retention:** Use `data retention <https://docs.mattermost.com/comply/data-retention-policy.html>`__ to automatically erase data after a set period of time, a feature that meets the Right to Erasure principle. In Team Edition, you can use database scripts to achieve the same result.
+- **Profile Deletion:** Delete a user’s personal information via `a command line tool <https://docs.mattermost.com/manage/command-line-tools.html>`__. The command permanently deletes all user information including messages created by the user.
```
Comment (cwarnermm) [inline source/about/certifications-and-compliance.rst:59]:
```suggestion
- **Profile Deletion:** Delete a user’s personal information via `a command line tool <https://docs.mattermost.com/manage/command-line-tools.html#mattermost-user-delete>`__. The command permanently deletes all user information including messages created by the user.
```

Comment (cwarnermm) [reply source/about/certifications-and-compliance.rst:59]: I've provided the updated URL.

**Review by cwarnermm — COMMENTED** [review]
Thanks for your docs contribution! I've provided one updated URL inline.

**Comment by ArmanChand** [comment]
@cwarnermm kindly merge this PR

**Review by justinegeffen — APPROVED** [review]

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**